.github/workflows/push-master.yml

```diff
@@ -4,17 +4,23 @@ on:
   push:
     branches:
       - master
+  pull_request:
+    branches:
+      - master
```

Comment:
It was, however, I was wondering if I should build on pull requests? It's just an idea - but I merged my PR to master and it failed to build. So I wondered what it would be like if it ran all jobs, and did a dry run on pull requests. Keen to hear what you think